addons/website_links/static/src/js/website_links.js

```diff
@@ -388,21 +364,43 @@ publicWidget.registry.websiteLinks = publicWidget.Widget.extend({
         'submit #o_website_links_link_tracker_form': '_onFormSubmit',
     },
 
+    init() {
+        this._super(...arguments);
+        this.selectMenu = {};
+    },
+
     /**
      * @override
      */
     start: async function () {
         var defs = [this._super.apply(this, arguments)];
 
-        // UTMS selects widgets
-        const campaignSelect = new SelectBox(this, "utm.campaign", _t("e.g. June Sale, Paris Roadshow, ..."));
-        defs.push(campaignSelect.attachTo($('#campaign-select')));
-
-        const mediumSelect = new SelectBox(this, "utm.medium", _t("e.g. InMails, Ads, Social, ..."));
-        defs.push(mediumSelect.attachTo($('#channel-select')));
+        async function attachSelectComponent(model, placeholderText, el) {
+            const props = {
+                placeholder: _t(placeholderText),
```

Comment:
It won't work, `_t` should directly receive the string literal to register it.
`placeholderText` has to be the translated text.